app/app/(music)/playlist/[id]/PlaylistView.tsx

```diff
@@ -38,15 +38,44 @@ const PlaylistView = ({
         playbuttonContext: playlist.uri,
       }}
     />
-    <h4 className="font-black uppercase pb-2">Average Playlist Statistics</h4>
-    <AudioFeatures audioFeatures={averageAudioFeatures} />
-    <div className="divider" />
-    <OffsetTrackList
-      contextUri={playlist.uri}
-      tracks={tracks}
-      showNumber
-      showAlbum
-    />
+    {playlist.tracks.total !== tracks.length && (
```

Comment:
Actually, this is invalid for large playlists. Can you make a new prop `hasPodcasts` that triggers this, and in the getData have a flag e.g. `let hasPodcast = false` that flips to true if in the filter it finds a podcast?